pkg/parser/model/model.go

```diff
@@ -1786,6 +1786,11 @@ type TableItemID struct {
 	IsSyncLoadFailed bool
 }
 
+// Key is used to generate unique key for TableItemID to use in the syncload
+func (t TableItemID) Key() string {
+	return fmt.Sprintf("%d%d%t%t", t.ID, t.TableID, t.IsIndex, t.IsSyncLoadFailed)
```

Comment:
Use some char like `#` or `-` as the separator, both `t.id=1&t.tableid=11` and `t.id=11&t.tableid=1` will map to `111`.
And we don't need the `is sync load failed` as the key. But we need one more field to tell whether it's a meta load or a full load.

Reply:
Updated.